src/modules/launcher/Wox.Plugin/ActionContextResult.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Wox.Plugin
+{
+    public class ActionContextResult
```

Comment:
What is this for? If this fixes the bug with PT Run not closing on clicking context menu items, then we can reference the corresponding issue.

Reply:
I think this was a test, removing this.